mu4/scenes/palette/qml/MuseScore/Palette/PaletteTree.qml

```diff
@@ -172,7 +172,7 @@ ListView {
     ScrollBar.vertical: ScrollBar {
         id: scrollbar
 
-        readonly property color baseColor: (ui.theme.backgroundColor.hslLightness > 0.5) ? "#28282a" : "#d7d7d5"
+        readonly property color baseColor: (ui.theme.backgroundPrimaryColor.hslLightness > 0.5) ? "#28282a" : "#d7d7d5"
```

Comment:
Can we replace these codes with colors from the theme?

Reply:
the palettes colouring was totally broken since we've integrated it in MU4, so it should be better to fix it separately, because now I've no idea which exact colours to use here